src/index.js

```diff
@@ -166,7 +156,6 @@ class UglifyJsPlugin {
       }
 
       compilation.plugin('optimize-chunk-assets', (chunks, callback) => {
-        const uglifiedAssets = new WeakSet();
```

Comment:
Hoisted this to `apply` scope. This was my change originally and was an optimization that I wanted to include but was late to do so before it got merged. I've tested it pretty thoroughly.